core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteractionEditorDirective.js

```diff
@@ -140,6 +145,10 @@ oppia.directive('stateInteractionEditor', [
             stateContentService.displayed.setHtml(
               DEFAULT_TERMINAL_STATE_CONTENT);
             stateContentService.saveDisplayedValue();
+            $scope.onSaveStateContent()(
+              stateContentService.stateName,
```

Comment:
Ideally, you would not need to pass stateName out. The client should have context. (See above.)

Reply:
Done.